src/CLICommand/parameters_helper.test.ts

```diff
@@ -364,4 +369,12 @@ describe('ParametersHelper class', () => {
 			CLICommand.parse(program, [...baseArgv, testCommandName]);
 		});
 	});
+
+	describe('getMaxDepth method', () => {
+		it(`Defaults to zero`);
+		it(`Does not accept a decimal`);
+		it(`Does not accept a negative integer`);
+		it(`Max depth is infinity when --all is specified`);
```

Comment:
Don't use infinity. In fact we should error if the user provides infinity and test for that.

Reply:
Actually it looks like Number.isInteger fails for POSITIVE_INFINITY